roles/ipareplica/files/py3test.py

```diff
@@ -2,9 +2,13 @@
 
 # Test ipaerver python3 binding
 try:
-    from ipaserver.install.server.replicainstall import install_check
+    from ipaserver.install.server.replicainstall import (  # noqa: F401
```

Comment:
Here you still have the black changes.
We can remove `py3test.py ` from all roles as the files are not used any more.

Reply:
Those are not black changes. Those are mine in order to add the noqa and don't break the line length.

Should I remove in this PR? I can add a commit just for that.

Reply:
a separate PR would be good - also the commented out code in the roles need to be removed.